### ISSUE TITLE
1.3.5 sdist tarball contains a lot of junk files

### ISSUE THREAD

**Issue by mgorny** [python-caldav/caldav#322]
The dist tarball contains a lot of files that don't seem to be long there. These include temporary files:

```
caldav-1.3.5/tests/#conf_private.py#
caldav-1.3.5/tests/#tmp_226.py#
caldav-1.3.5/tests/tmp_226~
```

`.pyc` files (leftovers from testing with different versions of pytest apparently), `.orig` and `.rej` files, backup files.

**Comment by tobixen** [comment]
ouch

**Comment by tobixen** [comment]
Whatever cookbook recipe I've been using for making pypi-releases, it forgot mentioning that it has to be done from a clean repository.  Released 1.3.6 now from a clean repository.
